mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -14,8 +14,11 @@ data:
   # test-options feature options.
   test-options.required-retest-contexts: "\"Jenkins GCE e2e\",\"Jenkins unit/integration\",\"Jenkins verification\",\"Jenkins GCE Node e2e\",\"Jenkins Kubemark GCE e2e\",\"Jenkins GCE etcd3 e2e\""
 
+  submit-queue.protected-branches: "master"
+  submit-queue.protected-branches-extra-contexts: "cla/linuxfoundation"
```

Comment:
Currently it has only "write" privileges. @thockin, ok to grant k8s-merge-robot admin in the kubernetes-org (or the repos with the merge-bot active?)